tests/sizers/wrapsizer.cpp

```diff
@@ -28,74 +28,85 @@
 // test class
 // ----------------------------------------------------------------------------
 
-class WrapSizerTestCase : public CppUnit::TestCase
+class WrapSizerTestCase
```

Comment:
Sorry, I just don't understand what are you trying to do with this class. The idea of having a common test fixture like this is to reuse the same setup code for multiple tests, but here you seem to be using only `m_leftWin` in one test and only `m_rightWin` in the other one. What sense does it make to do it like this?

Either we should use the same `m_win` in both tests (as we did before for the single test) or maybe we shouldn't have any fixture class at all if there is just nothing to reuse... Or, finally, if the current approach is really useful, there should be a comment here explaining why is it so because this is not obvious at all from just looking at the code.